client/dashboard/me/monetize-subscriptions/monetize-subscription.tsx

```diff
@@ -0,0 +1,307 @@
+import {
+	monetizeSubscriptionDisableAutoRenew,
+	monetizeSubscriptionQuery,
+	monetizeSubscriptionResumeAutoRenew,
+	monetizeSubscriptionStop,
+} from '@automattic/api-queries';
+import { formatCurrency } from '@automattic/number-formatters';
+import { CALYPSO_CONTACT } from '@automattic/urls';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { useNavigate, useRouter } from '@tanstack/react-router';
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalSpacer as Spacer,
+	Card,
+	CardHeader,
+	Button,
+	Notice,
+	CardBody,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { __, isRTL, sprintf } from '@wordpress/i18n';
+import { Icon, globe, chevronRight, chevronLeft } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+import { formatDate } from 'date-fns';
+import { monetizeSubscriptionRoute } from '../../app/router/me';
+import ActionList from '../../components/action-list';
+import { addFlashMessage } from '../../components/flash-message';
+import { PageHeader } from '../../components/page-header';
+import PageLayout from '../../components/page-layout';
+import {
+	getMonetizeSubscriptionsUrl,
+	getMonetizeSubscriptionsPageTitle,
+} from '../../me/monetize-subscriptions/urls';
+import type { MonetizeSubscription } from '@automattic/api-core';
+
+function MonetizeSiteHeader( { name, domain }: { name: string; domain: string } ) {
+	return (
+		<CardHeader>
+			<HStack spacing={ 5 } alignment="left">
+				<VStack alignment="center">
+					<Icon icon={ globe } />
+				</VStack>
+				<VStack>
+					<b>{ name }</b>
+					<div>{ domain }</div>
```

Comment:
nit: maybe we can use the [Text](https://github.com/Automattic/wp-calypso/tree/trunk/client/dashboard/components/text) component from the HD on these two:

```suggestion
					<Text weight="700">{ name }</Text>
					<Text>{ domain }</Text>
```

Reply:
It can be imported from:

```
import { Text } from '../../components/text';
```

Reply:
Done in 15fdc7c17f9b1a5e91acb9ab4f13ec029d131641